ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -63,10 +67,12 @@ private fun Footer(
     Row(
         modifier = childModifier
             .fillMaxWidth()
-            .height(intrinsicSize = IntrinsicSize.Max)
+            .height(intrinsicSize = IntrinsicSize.Min)
+            .requiredWidth(intrinsicSize = IntrinsicSize.Min)
             .padding(
                 start = UIConstant.defaultHorizontalPadding,
                 end = UIConstant.defaultHorizontalPadding,
+                top = UIConstant.defaultVerticalSpacing,
```

Comment:
NABD, but you could change to use `horizontal` and `vertical` parameters now.

Reply:
Changed